README.md

```diff
@@ -1 +1,126 @@
-# Math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove the template logo. You can replace with a personal logo if you have one.